sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityFactorModelReader.java

```diff
@@ -34,7 +34,7 @@ public void read(Handler handler) {
         for (SensitivityFactor factor : factors) {
             String functionId = factor.getFunctionId();
             if (factor.getFunctionType() == SensitivityFunctionType.BUS_VOLTAGE) {
-                Bus bus = new IdBasedBusRef(factor.getFunctionId()).resolve(network, TopologyLevel.BUS_BRANCH)
+                Bus bus = new IdBasedBusRef(factor.getFunctionId()).resolve(network, TopologyLevel.BUS_BREAKER)
```

Comment:
I really don't know what is the solution so I just propose that because inside the implementation I can retrieve the bus of the bus/view from the bus of the bus/breaker view. I am really not expert on powsybl-core buses, so what do you suggest?

Reply:
I really don't know :D 
But it seems more important to leave the choice to the user, than to leave the choice to the implementation.

So maybe we should find a way to let the user decide what he uses as input, and if he uses buses (bus view), the price for him will be that sometimes (when the bus is modified by an action or a contingency) he would get either an exception or a kind of "invalid" status ?

Reply:
For the moment, it will what I am doing: I throw an exception when it is not compatible. So finally, if I understand you well the solution is to remove totally the `IdBasedBusRef` part in the code and just take the id, given by the user, and pass it to the implementation?